patches/minecraft/net/minecraft/world/biome/BiomeDecorator.java.patch

```diff
@@ -4,82 +4,82 @@
  
      protected void func_150513_a(Biome p_150513_1_, World p_150513_2_, Random p_150513_3_)
      {
-+        net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.terraingen.DecorateBiomeEvent.Pre(p_150513_2_, p_150513_3_, field_180294_c));
++        net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.terraingen.DecorateBiomeEvent.Pre(p_150513_2_, p_150513_3_, field_180294_c, field_180294_c));
```

Comment:
These will break binary compat -- maybe make an overloaded method/constructor with the old name that calculates the `ChunkPos` from the `BlockPos`?

Reply:
Woops